source/_components/vacuum.xiaomi_miio.markdown

```diff
@@ -126,29 +126,12 @@ To fetch the token follow these instructions depending on your mobile phone plat
 #### {% linkable_title Windows and Android %}
 
 1. Configure the robot with the Mi-Home app.
-2. Enable developer mode and USB debugging on the Android phone and plug it into the computer.
-3. Get and install the [ADB tool for Windows](https://developer.android.com/studio/releases/platform-tools.html).
-4. Create a backup of the application `com.xiaomi.smarthome`:
-```bash
-$ adb backup -noapk com.xiaomi.smarthome -f backup.ab
-```
-5. If you have this message: "More than one device or emulator", use this command to list all devices:
-```bash
-$ adb devices
-```
-and execute this command:
-```bash
-$ adb -s DEVICEID backup -noapk com.xiaomi.smarthome -f backup.ab # (with DEVICEID the device id from the previous command)
-```
-6. On the phone, you must confirm the backup. DO NOT enter any password and press button to make the backup.
-7. Get and install [ADB Backup Extractor](https://sourceforge.net/projects/adbextractor/).
-8. Extract All files from the backup:
-```bash
-$ java.exe -jar ../android-backup-extractor/abe.jar unpack backup.ab backup.tar ""
-```
-9. Unzip the ".tar" file.
-10. Open the SQLite database `miio2.db` with a tool like SQLite Manager extension for FireFox.
-11. Get the token from "devicerecord" table.
+2. Get and extract the MiToolKit.zip(https://github.com/ultrara1n/MiToolkit/releases).
```

Comment:
Get and extract the [MiToolKit.zip](https://github.com/ultrara1n/MiToolkit/releases).

Reply:
Hey @arsaboo this means you've already fixed the mistake correct? This is my first PR with such a correction and I've just noticed the "Fix/Formatting/Language/Spelling" label.

Reply:
No, I have not. I just gave you an example of how it should look.